flink-runtime/src/main/java/org/apache/flink/runtime/rest/messages/job/JobDetailsInfo.java

```diff
@@ -165,6 +165,55 @@ public int hashCode() {
 		return Objects.hash(jobId, name, isStoppable, jobStatus, startTime, endTime, duration, now, timestamps, jobVertexInfos, jobVerticesPerState, jsonPlan);
 	}
 
+	public JobID getJobId() {
+		return jobId;
+	}
+
+	public String getName() {
+		return name;
+	}
+
+	@JsonProperty(FIELD_NAME_IS_STOPPABLE)
```

Comment:
you should do this for all getters, as otherwise we can run into the same issue should we ever rename the field.